### DISCUSSION TITLE
Automated Helm Release updates in Git?

### DISCUSSION THREAD

**Discussion by cer** [fluxcd/flux2#2978] [Q&A] [answered]
I am wondering if there is a plan for FluxCD to (1) scan Helm Repositories for new versions of chart (2) git commit/push the new version (3) and `helm upgrade`.

**Comment by stefanprodan** [comment, marked as answer]
This is supported for Helm charts stored in container registries as OCI artifacts.  It works in the same as with app image automation, you just place the update marker in your HelmRelease YAML next to the chart version.

**Reply by cer** [reply]
That's good to know.

Any plans to do that for regular Helm repositories?

**Reply by stefanprodan** [reply]
No plans for that, I really hope the Helm community moves off http repos which have major scaling and performance issues. We’ll have to wait and see if Helm v4 drops http repos, that will force the community to adopt OCI.

**Reply by cer** [reply]
> This is supported for Helm charts stored in container registries as OCI artifacts. It works in the same as with app image automation, you just place the update marker in your HelmRelease YAML next to the chart version.

This is an awesome feature!

**Reply by onedr0p** [reply]
You can also hook Renovate up to your repository, it supports tracking Flux resources like `HelmRelease` and others.

https://docs.renovatebot.com/modules/manager/flux/
